devel/gn-devel/Portfile

```diff
@@ -34,7 +34,7 @@ configure.pre_args
 
 build.dir           ${worksrcpath}/out
 build.cmd           ${prefix}/bin/ninja
-build.target
+build.target        gn
```

Comment:
I presume changing the build target has no impact on installed files?

Reply:
Correct, the only installed file is the `gn` binary, and the `gn` target is enough to build it. So unless someone is running `port test`, the `gn_unittest` target is needlessly built as well. I guess the unusual thing here is that if a target isn’t specified, then ninja builds all targets, unlike make which would only build some default target.